src/ai/composite/component.cpp

```diff
@@ -19,19 +19,15 @@
  */
 
 #include "ai/composite/component.hpp"
-#include "ai/composite/engine.hpp"
 #include "ai/composite/property_handler.hpp"
 #include "config.hpp"
 #include "log.hpp"
-#include "units/unit.hpp"
 
-#include "ai/formula/ai.hpp"
 
 #include <boost/regex.hpp>
 
 namespace pathfind {
 
-struct pathfind;
```

Comment:
Here's another one where the namespace can be removed.

Reply:
> Here's another one where the namespace can be removed.

I’m not quite sure what to do, so I won’t proceed with the merger.